### DISCUSSION TITLE
Can you recover data from a backup without system installed from a cifs / smb drive?

### DISCUSSION THREAD

**Discussion by fidesachates** [longhorn/longhorn#7554] [General]
I have my backups configured to a CIFS / SMB drive. Having read [this](https://longhorn.io/docs/1.5.3/advanced-resources/data-recovery/recover-without-system/), I notice it says only s3 and nfs. Is this just an oversight in documentation and providing a cifs / smb url would work? I assume not because there's no mention of how to pass in the CIFS / SMB credentials. 

Should I raise a suggestion for this feature? Would even like it if it could read from a local path instead of remote drives since one can always manually mount the cifs / smb.

**Comment by c3y1huang** [comment]
I think this might have been overlooked during the feature implemention. @fidesachates , could you help to create a feature request for this?

**Comment by fidesachates** [comment]
@c3y1huang done. https://github.com/longhorn/longhorn/issues/7921
